app/lib/three_scale/bot_protection/base.rb

```diff
@@ -3,12 +3,20 @@
 module ThreeScale
   module BotProtection
     module Base
+      private
+
       def bot_protection_level
         site_account.settings.spam_protection_level
       end
 
       def bot_protection_enabled?
-        Recaptcha.captcha_configured? && bot_protection_level != :none
+        Recaptcha.captcha_configured? && bot_protection_level != :none && !request_from_auth_provider?
+      end
+
+      def request_from_auth_provider?
```

Comment:
is this a valid check for all classes that include this module?

Reply:
That's old code. You reviewed too fast!